OracleHTTPServer/dockerfiles/14.1.2.0.0/container-scripts/helloWorld.html

```diff
@@ -0,0 +1,12 @@
+<!DOCTYPE html PUBLIC "-//IETF//DTD HTML 2.0//EN">
```

Comment:
Does this file need a copyright?

Reply:
Normally we do not put copyright in html